tests/archive/test_fertility.py

```diff
@@ -29,7 +29,7 @@ def cleanup_debug_files(self):
             if sc.path(f).exists():
                 os.remove(f)
 
-    def sweep_seed(self, seeds=None, pars=fp.pars(), # CK: this is a mutable object so shouldn't be defined here
+    def sweep_seed(self, seeds=None, pars=None, # CK: this is a mutable object so shouldn't be defined here
```

Comment:
Can we just delete all the files in this folder? Creates a maintenance burden for over-eager AIs that descend into subfolders :)

Reply:
https://github.com/fpsim/fpsim/issues/623